src/index.html

```diff
@@ -1,17 +1,171 @@
+        <input
+        class="input"
+        type="date"
+        id="BirthDate"
+        name="BirthDay"
+        required
+        >
+        <br><label for="accept">I accept the term of the agreement</label>
+        <input
+        type="checkbox"
+        id="accept"
+        name="accept"
+        required
+        >
+
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label for="E-mail">E-mail: </label>
+        <input
+        class="input"
+        type="email"
+        name="E-mail"
+        id="E-mail"
+        placeholder="email@example.com"
+        required
+        >
+        <br><label for="Password">Password:</label>
+        <input
+        type="password"
+        id="Password"
+        name="Password"
+        required
+        >
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label for="Yes">Do you love cats?</label>
+
+        <input
+          class="fieldset__input"
+          type="radio"
+          name="terms"
+          id="Yes"
+          required
+        >
+        <label for="Yes">Yes</label>
+        <input
+          class="fieldset__input"
+          type="radio"
+          name="terms"
+          id="No"
+        >
+        <label for="No"> No</label>
+
+        <br><label for="color">What is your favorite color?
+          <input
+            class="label__color"
+            type="color"
+            name="color"
+            id="color"
+            required
+          >
+            </label>
+
+
+        <br><label for="time">What time do you go to bed?</label>
+        <input
+          class="input"
+          type="time"
+          step="2"
+          id="time"
+          name="time"
+          min="22:20"
+          max="00:00"
+          required
+        >
+
+        <br><label for="brandsOfcar" >What are your favorite brand of cars?</label>
+        <select
+          name="brand"
+          id="brandsOfcar"
+          required
+          multiple
+          >
+          <option value="BMW">BMW</option>
+          <option value="Audi">Audi</option>
+          <option value="Lada">Lada</option>
+        </select>
+
+        <br><label for="scrol_volume">How do you rate our work?</label>
+        <input
+        class="scrol_volume"
+        type="range"
+        id="scrol_volume"
+        name="scrol_volume"
+        value="0"
+        minlength="0"
+        maxlength="100"
+        >
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label for="textArea">Comments:</label>
+        <textarea
+            class="input"
+            name="textArea"
+            id="textArea"
+            cols="-3"
+            rows="-1"
+       ></textarea>
+
+        <br><label for="RecomendBloc">Would you recommend us?</label>
+        <select
+          name="RecomendBloc"
+          id="RecomendBloc"
+          required
+          >
```

Comment:
```suggestion
        <select
          name="RecomendBloc"
          id="RecomendBloc"
          required
        >
```
format your code according to the checklist